zipkin-storage/zipkin2_cassandra/src/test/resources/trace_by_service_span-stress.yaml

```diff
@@ -71,8 +71,8 @@ insert:
 #
 queries:
    select:
-    cql: SELECT * FROM trace_by_service_span WHERE service_name = ? AND span_name = ? AND bucket = ? LIMIT 1
+    cql: SELECT * FROM trace_by_service_span WHERE service = ? AND span = ? AND bucket = ? LIMIT 1
     fields: samerow
    by_duration:
-    cql: SELECT * FROM trace_by_service_span WHERE service_name = ? AND span_name = ? AND bucket = ? AND duration < ? LIMIT 1
-    fields: samerow
\ No newline at end of file
+    cql: SELECT * FROM trace_by_service_span WHERE service = ? AND span = ? AND bucket = ? AND duration < ? LIMIT 1
+    fields: samerow
```

Comment:
it looks like we're missing two queries here:
 - search by timestamp range,
 - search by timestamp range and duration

Reply:
I couldn't quickly find out how to do a range query in a stress test, so left it out for now with a TODO